common/changes/@rushstack/ts-command-line/keco-tslint-microsoft-contrib_2020-03-28-03-28.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/ts-command-line",
+      "comment": "Snapshots updated",
```

Comment:
`Snapshots update` isn't something that should go in a CHANGELOG.  It doesn't affect the shipped code.

Also I'm slightly puzzled that Rush created a changefile with type `none` AND a message. (?)

Reply:
I manually modified this after being puzzled by the snapshot churn. I agree regarding your first point.